tests/LSP17ContractExtension/LSP17Extendable.behaviour.ts

```diff
@@ -723,9 +709,82 @@ export const shouldBehaveLikeLSP17 = (
             );
         });
         it("should pass even if there is an extension for it that reverts", async () => {
-          await context.accounts[0].sendTransaction({
-            to: context.contract.address,
-            data: "0x01",
+          await expect(
+            context.accounts[0].sendTransaction({
+              to: context.contract.address,
+              data: "0x01",
+            })
+          ).to.not.be.reverted;
+        });
+      });
+
+      describe("when calling with a payload preprended with 4 bytes of 0", () => {
+        describe("when the payload is equal to 4 bytes of zeros", () => {
+          describe("with sending value", () => {
+            it("should pass and emit ValueReceived value", async () => {
+              const amountSent = 2;
+              await expect(
+                context.accounts[0].sendTransaction({
+                  to: context.contract.address,
+                  data: "0x00000000",
+                  value: amountSent,
+                })
+              )
+                .to.emit(context.contract, "ValueReceived")
+                .withArgs(context.accounts[0].address, amountSent);
+            });
+          });
+          describe("without sending value", () => {
+            it("should pass", async () => {
+              await expect(
+                context.accounts[0].sendTransaction({
+                  to: context.contract.address,
+                  data: "0x00000000",
+                })
+              ).to.not.be.reverted;
+            });
+          });
+        });
+        describe("when the payload is larger than 4 bytes of zeros", () => {
```

Comment:
Suggested improvement in the description of the test

```suggestion
        describe("when the payload is `0x00000000` + some random data ('graffiti')", () => {
```